pkg/cmd/build/build.go

```diff
@@ -71,5 +74,9 @@ func Run(ctx context.Context, namespace, buildKitHost string, isOktetoCluster bo
 		analytics.TrackBuildTransientError(buildKitHost, success)
 		return err
 	}
+	if strings.Contains(err.Error(), "insufficient_scope: authorization failed") {
+		err = okErrors.UserError{E: fmt.Errorf("Authorization failed."),
```

Comment:
could we extract the image that we tried to build and say something like: `You are not authorized to push image foo/bar:tag`  ?